### PR TITLE
Simplify Hero

### PR DESCRIPTION
**Pull request by sampoder**
my proposal:

Screenshot 2023-09-13 at 12 13 25 AM

### CODE REVIEW

File: public/ddoski/3Rw3Uq_oxDVdJvZoqSWvAtKYYd_Uijdh2XRVIgV3ST6_fA2yFcmTzroO_n2NfuSO7Y_F49Sl3zgmoxLsUic1HGLty7ah8pFTI6OQUXG2jDIALqMW9lLwgoOit8FrpJmw-xJUHFe9jZQ894PQHZK9ckOsag=s2048.png
Comment (matthewlee626) [inline public/ddoski/3Rw3Uq_oxDVdJvZoqSWvAtKYYd_Uijdh2XRVIgV3ST6_fA2yFcmTzroO_n2NfuSO7Y_F49Sl3zgmoxLsUic1HGLty7ah8pFTI6OQUXG2jDIALqMW9lLwgoOit8FrpJmw-xJUHFe9jZQ894PQHZK9ckOsag=s2048.png:1]: so lit

File: pages/index.tsx [hunk 1]
```diff
@@ -5,6 +5,10 @@ import Image from "next/image";
 import JSConfetti from "js-confetti";
 import { useRef, useEffect, use } from "react";
 
+let images = [
```
Comment (matthewlee626) [inline pages/index.tsx:8]: later, lets excise this into another file

File: pages/index.tsx [hunk 2]
```diff
@@ -17,7 +21,7 @@ export default function Home() {
       className='overflow-hidden min-h-screen text-center font-sans text-white bg-cover'
       style={{ backgroundImage: "url(/bg.webp)" }}>
       <nav className='text-xl md:text-3xl w-full drop-shadow-lg flex justify-center md:justify-between items-center mb-8 font-header text-left px-4 py-3 font-semibold'>
-        <span>Cal Hacks x MindsDB</span>
+        <img src={images[Math.floor(Math.random()*images.length)]} className="drop-shadow-lg" style={{height: '48px'}} />
```
Comment (matthewlee626) [inline pages/index.tsx]: Forgive me if I'm wrong, but won't this build once on server; thus, the same bear gets loaded each time?

Comment (sampoder) [reply pages/index.tsx]: yeah you're right, i realised this after making the PR... i'll make some other alternative tomorrow

File: pages/index.tsx (hunk 1 above)
Comment (sampoder) [reply pages/index.tsx:8]: agreed, another thing to do is make the file names not.... horrid

Comment (matthewlee626) [reply pages/index.tsx:8]: i love index

File: pages/index.tsx (hunk 2 above)
Comment (matthewlee626) [reply pages/index.tsx]: theres probably some dumb way like a SSR with a little fetch which just returns the image url lol

Comment (aryanvichare) [reply pages/index.tsx]: Might want to explore using this - https://nextjs.org/docs/pages/building-your-application/data-fetching/get-server-side-props

**Pull request merged**